tests/acceptance/bootstrap/OcmContext.php

```diff
@@ -53,18 +52,6 @@ public function before(BeforeScenarioScope $scope): void {
 		$environment = $scope->getEnvironment();
 		// Get all the contexts you need in this context from here
 		$this->featureContext = $environment->getContext('FeatureContext');
-		$this->notificationContext = $environment->getContext('NotificationContext');
-	}
-
-	/**
-	 * Delete all the inbucket emails
-	 *
-	 * @AfterScenario @email
-	 *
-	 * @return void
-	 */
-	public function clearInbucketMessages():void {
-		$this->notificationContext->clearInbucketMessages();
```

Comment:
this step was deleted inbucket message only after tests/acceptance/features/apiOcm/createInvitation.feature:138 

deleting emails after every Ocm test I think is unnecessary. A simpler way is to use a not frequently used user, like David -> this will reduce the number of mails for him and the test will pass for sure